### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by rajadain**
Connects #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,206 @@
...
 
 ```bash
-$ git clone https://github.com/azavea/usace-flood-geoprocessing.git
-$ cd usace-flood-geoprocessing
-$ ./sbt assembly
+$ ./sbt "project ingest" assembly
 ```
 
-To use a Docker based Scala build environment, you can use:
+which should result in `ingest/target/scala-2.10/usaceflood-ingest-assembly-0.1.0-SNAPSHOT.jar`.
```
Comment (lewfish) [inline README.md]: I actually have things set up in https://github.com/lewfish/usace-flood-geoprocessing/blob/feature/travis/project/Version.scala so the file name will be `usaceflood-ingest-assembly-SNAPSHOT.jar`

Comment (rajadain) [inline README.md]: Ah, thanks! Will update. We will probably also have to update here: https://github.com/azavea/usace-flood-model/blob/develop/deployment/ansible/roles/usace.geoprocessing/defaults/main.yml#L4

File: README.md [hunk 2]
```diff
@@ -1,68 +1,206 @@
...
 
-If you are using docker-machine to host the docker container, then be sure to use the correct IP address in the above curl statements:
+The actual server is run by executing the following:
 
 ```bash
-$ export DOCKER_IP=`docker-machine ip $DOCKER_MACHINE_NAME`
-$ curl --silent \
-    --data-binary @summary/target/scala-2.10/usaceflood-geoprocessing-assembly-0.0.1.jar \
-    "http://$DOCKER_IP:8090/jars/geoprocessing"
+$ spark-submit server/target/scala-2.10/usaceflood-server-assembly-0.1.0-SNAPSHOT.jar
```
Comment (lewfish) [inline README.md]: These look like instructions for running on your host. Consider mentioning where your data has to be for this to work.

File: README.md [hunk 3]
```diff
@@ -1,68 +1,206 @@
...
-## Usage
+ 1. [`ingest`](#ingest): For taking input elevation data and processing it into a format usable by [GeoTrellis](https://github.com/geotrellis/geotrellis)
+ 2. [`server`](#server): A [Spray](https://github.com/spray/spray) based web service that performs geoprocessing operations
 
-First, build the assembly JAR for this project:
+Both projects are run using [Apache Spark](http://spark.apache.org/).
+
+## `ingest`
+
+This project takes input elevation data and processes it into RDDs that will be ready by GeoTrellis. The input data was originally in DEM files supplied by the client. Using [GDal](http://www.gdal.org/), these were stitched together, then clipped to the relevant area of interest, and tiled (see [azavea/usace-flood-model#2](https://github.com/azavea/usace-flood-model/issues/2)).
```
Comment (lewfish) [inline README.md]: 'ready' -> 'ready to use'

**Comment by lewfish** [comment]
+1

**Comment by rajadain** [comment]
Addressed all comments. Merging shortly. Thanks for taking a look!

**Pull request merged**